src/EnergyPlus/api/runtime.h

```diff
@@ -74,6 +75,7 @@ ENERGYPLUSLIB_API void callbackEndOfSystemSizing(EnergyPlusState state, std::fun
 ENERGYPLUSLIB_API void callbackEndOfAfterComponentGetInput(EnergyPlusState state, std::function<void(EnergyPlusState)> f);
 ENERGYPLUSLIB_API void callbackUnitarySystemSizing(EnergyPlusState state, const std::function<void(EnergyPlusState)> &f);
 ENERGYPLUSLIB_API void registerStdOutCallback(EnergyPlusState state, std::function<void(const std::string &)>);
+ENERGYPLUSLIB_API void registerProgressCallback(EnergyPlusState state, std::function<void(int const)>);
```

Comment:
Add a C++ compatible overload for registerProgressCallback that was missing

Reply:
:+1: